src/types/wrappers/MinimizeAdjustment.jl

```diff
@@ -0,0 +1,30 @@
+
+"""
+$(TYPEDEF)
+
+A wrapper that adds a quadratic objective that minimizes total squared error
+("adjustment") from a certain reference variable assignment ("reference flux").
+
+Length of `reference_assignment` must be the same as the the number of
+variables in the inner model.
+
+This is used to implement [`minimize_metabolic_adjustment`](@ref).
+
+# Example
+```
+m = load_model("e_coli_core.xml")
+adjusted_model = m |> MinimizeAdjustmentModel(fill(0.1, n_variables(m)))
+```
+"""
+struct MinimizeAdjustmentModel <: AbstractModelWrapper
+    inner::AbstractMetabolicModel
+    reference_assignment::Vector{Float64}
+end
+#TODO: sparse MinimizeAdjustmentModel with dict/sparseVec
+#TODO: MinimizeReactionAdjustmentModel?
+#TODO: MinimizeEnzymeAdjustmentModel?
```

Comment:
hmm it would be better if this were automatically handled, e.g. if you specify a variable in `reference_assignment` then it goes into the QP, otherwise it is ignored

Reply:
Yeah I assume this could work by specifying the semantics directly. Will see when I'm at it.